libnmstate/schemas/operational-state.yaml

```diff
@@ -716,6 +717,45 @@ definitions:
               type: string
           required:
             - peer
+
+  interface-wireguard:
+    rw:
+      properties:
+        type:
+          type: string
+          enum:
+            - wireguard
+        wireguard:
+          type: object
+          properties:
+            fwmark:
+              type: integer
```

Comment:
The wireguard wg CLI tool supports string like `0x0001`, please support it also.